core/src/price_finding/optimization_price_finder.rs

```diff
@@ -260,8 +261,9 @@ impl PriceFinding for OptimisationPriceFinder {
             // We are solving the batch before the current one
             let batch_id = (now.timestamp() / 300) - 1;
             let date = now.format("%Y-%m-%d");
+            let pwd = env::current_dir()?;
```

Comment:
naming-nit: I would prefer `wd` or even better `working_dir`. I think `pwd` stands for "Print Working Directory".